.github/workflows/build.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches:
       - main
-      - 'release/v**'
```

Comment:
I don't see a need for us to do both a regular build, and a release build on `release/v***` branches

Reply:
I suppose if one depended on the other, it would make sense, but not like this. However, pretty sure the builds are required in the PR before merging... Nothing we can't override obviously

Reply:
Nevermind, this is just for the push event, merge away!